Godeps/_workspace/src/k8s.io/kubernetes/pkg/api/validation/validation_test.go

```diff
@@ -4171,6 +4171,18 @@ func TestValidateSecurityContextConstraints(t *testing.T) {
 	negativePriority := validSCC()
 	negativePriority.Priority = &invalidPriority
 
+	requiredCapAddAndDrop := validSCC()
+	requiredCapAddAndDrop.RequiredCapabilities = api.Capabilities{
+		Add: []api.Capability{"foo"},
+		Drop: []api.Capability{"foo"},
+	}
+
+	allowedCapListedInRequiredDrop := validSCC()
+	allowedCapListedInRequiredDrop.RequiredCapabilities = api.Capabilities{
+		Drop: []api.Capability{"foo"},
+	}
+	allowedCapListedInRequiredDrop.AllowedCapabilities = []api.Capability{"foo"}
```

Comment:
Okay, I'll buy that for now.  If people find it incredibly confusing we can add a validation later.